functions/Get-DbaDatabaseFreespace.ps1

```diff
@@ -168,12 +206,16 @@ Returns database files and free space information for the db1 and db2 on localho
 				'UsedSpaceMB' = $row.UsedSpaceMB;`
 				'FreeSpaceMB' = $row.FreeSpaceMB;`
 				'FileSizeMB' = $row.FileSizeMB;`
-				'PercentUsed' = $row.PercentUSed
+				'PercentUsed' = $row.PercentUSed;`
+				'AutoGrowth' = $row.GrowthMB;`
+				'AutoGrowType' = $row.GrowthType;`
```

Comment:
Latest commit 50213e8 seems to have reverted GrowthType back to KB instead of MB?